solutions/web3-fetching-strategies/pages/_app.tsx

```diff
@@ -0,0 +1,18 @@
+import type { AppProps } from 'next/app'
+import type { LayoutProps } from '@vercel/examples-ui/layout'
+
+import { getLayout } from '@vercel/examples-ui'
+
+import '@vercel/examples-ui/globals.css'
+
+function App({ Component, pageProps }: AppProps) {
+  const Layout = getLayout<LayoutProps>(Component)
+
+  return (
+    <Layout title="web3-fetching-strategies" path="solutions/web3-fetching-strategies">
```

Comment:
Why is it called `web3-fetching-strategies`? what's the strategies part referring to?

Reply:
Renamed to web3-data-fetching. smaller scope but useful for web3 folks 👯